test/python/test_conv2d.py

```diff
@@ -0,0 +1,60 @@
+# ==============================================================================
+#  Copyright 2018-2020 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge fusedConv2D tests.
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+import platform
+
+import tensorflow as tf
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.ops import array_ops
+from common import NgraphTest
+from tensorflow.python.framework import dtypes
+
+import numpy as np
```

Comment:
```suggestion
import pytest
import platform
import numpy as np

import tensorflow as tf
from tensorflow.python.ops import nn_ops
from tensorflow.python.ops import array_ops
from tensorflow.python.framework import dtypes

from common import NgraphTest
```

Reply:
Minor: We have a order in which we have the includes etc : 
Standard
TF
nGraph
Bridge